docs/features/UNIFIED_CANVAS.md

```diff
@@ -0,0 +1,96 @@
+The Unified Canvas is a tool designed to facilitate composition, enabling artists to use all available generation modes in a single unified workflow. This documentation will help explain the basics of using the Unified Canvas, as well as some advanced tools available to power users of the canvas.
```

Comment:
I suggest adding an intro that lists the common functions that the Unified Canvas serves. It does three main things: inpainting, outpainting and creating a drawing for img2img. Say this explicitly and maybe give an example of a typical workflow.

Reply:
It is not restricted to these, as it can do txt2img and img2img as well. I also provide detailed examples later in the docs, so will leave this as explaining generation methods and use cases.